### PR TITLE
Feature-1 login and Signup Page

### PR DESCRIPTION
**Pull request by amitfegade121**
I have created login.html and sign.html pages.

### CODE REVIEW

File: login.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Login Form</title>
+  </head>
+  <body bgcolor="cyan">
```
Comment (kiranfegade121) [inline login.html]: Please change background color to sky blue.

**Review by kiranfegade121 — CHANGES_REQUESTED** [review]
Please do the changes and push those changes.

File: login.html (hunk 1 above)
Comment (amitfegade121) [reply login.html]: I have done the changes. Please review again...

**Review by kiranfegade121 — APPROVED** [review]
Okay.. Looks fine... Go ahead and merge the changes...

**Pull request merged**